view/theme/frio/templates/jot.tpl

```diff
@@ -162,19 +163,22 @@
 {{* The jot modal - We use a own modal for the jot and not the standard modal
 from the page template. This is because the special structure of the jot
 (e.g.jot navigation tabs in the modal titel area).
-The in the frio theme the jot will loaded regulary and is hidden by default.)
+Then in the frio theme the jot will loaded regulary and is hidden by default.)
 The js function jotShow() loads the jot into the modal. With this structure we
-can load different content into the jot moadl (e.g. the item edit jot)
+can load different content into the jot modal (e.g. the item edit jot)
 *}}
 <div id="jot-modal" class="modal fade" role="dialog">
 	<div class="modal-dialog">
 		<div id="jot-modal-content" class="modal-content"></div>
 	</div>
 </div>
 
-
 <script type="text/javascript">
 	$('iframe').load(function() {
 		this.style.height = this.contentWindow.document.body.offsetHeight + 'px';
 	});
 </script>
+
+<script>
+	dzFactory.setupDropzone('#jot-text-wrap', $('#profile-jot-text'), {{$max_imagesize}}); 
```

Comment:
```suggestion
	dzFactory.setupDropzone('#jot-text-wrap', '#profile-jot-text', {{$max_imagesize}}); 
```

Reply:
Removed this also in 
- view/templates/item/compose.tpl
- view/theme/frio/js/modal.js